subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -1331,28 +1351,15 @@ static int prop_set(struct net_buf_simple *buf,
 	/* Properties are always set in light actual representation: */
 	case BT_MESH_LIGHT_CTRL_PROP_LIGHTNESS_ON:
 		srv->cfg.light[LIGHT_CTRL_STATE_ON] = from_actual(micro / MICRO_PER_UNIT);
```

Comment:
Can this setting part be baked into the function call? It makes the API less prone to mistakes of the type where you update one cfg.light but mistakenly call `update_lightness_setpoint` with a different state. As in
```c
update_lightness_setpoint(srv, LIGHT_CTRL_STATE_ON, from_actual(micro / MICRO_PER_UNIT);
```

Reply:
Added :)